src/cmd/cli/command/commands.go

```diff
@@ -834,6 +837,18 @@ var deleteCmd = &cobra.Command{
 	},
 }
 
+var deploymentsCmd = &cobra.Command{
```

Comment:
feels like this command will get unwieldy fast, without a way to scope the list. Now wondering whether deployments is simply a subcommand of `defang project` #360 cf. `vercel project`.